cert-manager/cert-monitor.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+data:
+  prometheus.yaml: |
+    global:
+      scrape_interval:     600s
+      evaluation_interval: 600s
+    rule_files:
+      - "cert_rules.yaml"
+    scrape_configs:
+      - job_name: certmanager
+        static_configs:
+        - targets: ['cert-manager.cert-manager:9402']
+        metric_relabel_configs:
+        - source_labels: ['namespace']
+          target_label: 'k8s_ns'
+  cert_rules.yaml: |
+    groups:
+    - name: certificate_rules
+      rules:
+      - record: certificate_expire_remaining_days 
+        expr: (certmanager_certificate_expiration_timestamp_seconds - time())/86400
+  sdexporter.yaml: |
+    static_metadata:
+    - metric: certificate_expire_remaining_days
+      type: gauge 
+      value_type: double 
+      help: Seconds until a certificate days
+kind: ConfigMap
+metadata:
+  name: exporterconfig
+  namespace: cert-manager
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: prom-certmanager
+  name: prom-certmanager
+  namespace: cert-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prom-certmanager
+  template:
+    metadata:
+      labels:
+        app: prom-certmanager
+    spec:
+      containers:
```

Comment:
You want to specify resources for these, right?

Reply:
Yes! Also I do want to make this image run as non-root, just didn't had enough time to test it, but I'll test right now and just push the manifest corrected